client/src/pages/private/ParticipantTable.js

```diff
@@ -282,22 +257,21 @@ const ParticipantTable = () => {
       if (!columns) return;
       if (!selectedTab) return;
       setLoadingData(true);
-      const { data, pagination } = await fetchParticipants(
-        currentPage * pageSize,
-        reducerState.locationFilter,
-        reducerState.fsaFilter || '',
-        reducerState.lastNameFilter || '',
-        reducerState.emailFilter || '',
-        reducerState.order.field,
-        reducerState.order.direction,
-        reducerState.siteSelector,
+      const { data, pagination: newPagination } = await fetchParticipants(
+        pagination.page * pageSize,
+        filter.locationFilter?.value || '',
+        filter.fsaFilter?.value || '',
+        filter.lastNameFilter?.value || '',
+        filter.emailFilter?.value || '',
```

Comment:
Threre's still a fair bit of duplication here. I wonder if creating a prop type could be useful here.

Reply:
Totally, this gets alleviated a bit in https://github.com/bcgov/hcap/pull/617